README.md

```diff
@@ -8,11 +8,11 @@
 
 ## Overview
 
-**Impossible Traveller Threat Response Solution Pack** is designed to provide a set of investigation and utility playbooks to respond to Concurrent Successful Authentications To the Same Account From Multiple Countries. SIEM usually picks up and reports these events.
+**Impossible Traveler Threat Response** Solution Pack provides a set of investigation and utility playbooks to respond to concurrent successful authentications to the same account from multiple countries. A SIEM usually picks up and reports these events.
```

Comment:
Please keep the old spelling of "Traveller" with a *double l* to maintain consistency with the solution pack title and documentation.